bangazonapi/views/profile.py

```diff
@@ -277,12 +280,33 @@ def favoritesellers(self, request):
                 }
             ]
         """
-        customer = Customer.objects.get(user=request.auth.user)
-        favorites = Favorite.objects.filter(customer=customer)
+        if request.method == "GET":
+            customer = Customer.objects.get(user=request.auth.user)
+            favorites = Favorite.objects.filter(customer=customer)
+
+            serializer = FavoriteSerializer(
+                favorites, many=True, context={'request': request})
+            return Response(serializer.data)
+        
+        if request.method == "POST":
+            favorite = Favorite()
+
+            try:
+                favorite.customer = Customer.objects.get(user=request.auth.user)
+                favorite.seller = Customer.objects.get(pk=request.data["seller"])
+                serializer = FavoriteSerializer(
+                    data=favorite, context={'request': request})
+                if serializer.is_valid(raise_exception=True):
+                    favorite.save()
```

Comment:
Since you won't have a favorite instance, favorite can be changed to `serializer.save()` it will work the same way